### ISSUE TITLE
remove babel and extract-text-webpack-plugin from dependencies to devDependencies

### ISSUE THREAD

**Issue by juhaelee** [ealush/butter-toast#62]

**Comment by ealush** [comment]
Good catch. Must have missed it.
`babel-runtime` needs to be a dependency, but `extract-text-webpack-plugin` is definitely a devDependency.
Will push a fix soon.
Thanks! 👍

**Comment by ealush** [comment]
Published 2.1.2. Should be fixed now.

**Comment by juhaelee** [comment]
any reason why you need babel-runtime to be a dependency?

**Comment by ealush** [comment]
> any reason why you need babel-runtime to be a dependency?

Yes. I use babel `transform-runtime` as an alternative to babel-polyfill - which pollutes the global scope. Babel runtime specifically prevents duplication even in shared space.
See [here](https://babeljs.io/docs/plugins/transform-runtime/):
> This is where the transform-runtime plugin comes in: all of the helpers will reference the module babel-runtime to avoid duplication across your compiled output. The runtime will be compiled into your build.

> I also noticed you locked down react to a specific version as dev dependency:

Setting it as devDependency means that npm install won't install react in development, which is unwanted.
Currently this package supports React15. I might add `"react": "15.0.0"` as a peerDependency.
